gc/gc-iceberg/src/main/java/org/projectnessie/gc/iceberg/IcebergContentToFiles.java

```diff
@@ -207,7 +209,7 @@ static StorageUri checkUri(String type, String location, ContentReference conten
       return StorageUri.of("file://" + location);
     }
 
-    if (loc.isLocalFile()) {
+    if (SCHEME_FILE.equals(loc.scheme())) {
```

Comment:
Nit (non-blocking): I think, we can remove that constant in StorageUri. `"file"` here would be good enough